uk.co.compucorp.civicrm.hrcore/CRM/HRCore/CMSData/UserAccount/Drupal.php

```diff
@@ -71,4 +71,11 @@ public function isUserDisabled($contactData) {
 
     return !$user->status ? true : false;
   }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function findUserByEmail($mail) {
```

Comment:
Do we need this method here? since you added it to the `CRM_HRCore_Service_DrupalUserService` class also and its the `CRM_HRCore_Service_DrupalUserService` class that is used as the drupal service class in the `CreateUserRecordTaskForm` class.
I know the classes are similar and a bit confusing, we need to harmonize both eventually.